src/com/github/miachm/sods/OdsWritter.java

```diff
@@ -309,9 +310,38 @@ private void writeCellStyle(XMLStreamWriter out, Style style) throws XMLStreamEx
             out.writeAttribute("style:family", "table-cell");
             out.writeAttribute("style:name", key);
 
-            if (style.getBackgroundColor() != null) {
+            if (style.getBackgroundColor() != null || style.isBorder() || style.isBorderTop() 
+            		|| style.isBorderBottom() || style.isBorderLeft() || style.isBorderRight() || style.isWrap()) 
```

Comment:
Flow structures must be simple and straightforward to follow. Therefore, it's really important to minimise the number of different conditions you have in an if.

The rule of thumb is, you are doing something wrong if you have more than 2 OR/AND operators. In this case I would wrap these conditions in a different and unique function (for example style.usesBorders())